### PR TITLE
Enabled janitor_monkey Support

### PR DESCRIPTION
**Pull request by dthagard**
Updated from https://github.com/mlafeldt/docker-simianarmy/pull/1/files

### CODE REVIEW

**Comment by dthagard** [comment]
Sorry for the spam. I was not paying attention to the branch I was updating.

File: docs/configuration-properties.md [hunk 1]
```diff
@@ -81,7 +81,61 @@ See https://github.com/Netflix/SimianArmy/wiki/Chaos-Settings for a detailed des
 
 ## Janitor Monkey Properties
 
-Janitor Monkey is disabled and cannot be configured at the moment.
+| Key | Default |
+| --- | ------- |
+| simianarmy/janitor/enabled | true
```
Comment (mlafeldt) [inline docs/configuration-properties.md:86]: This is not "true". 😄

Comment (mlafeldt) [reply docs/configuration-properties.md:86]: Also, could you please add a trailing "|" in every row for consistency?

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1 @@
+.project
```
Comment (mlafeldt) [inline .gitignore:1]: What is this?

**Review by mlafeldt — COMMENTED** [review]
Thanks for the PR. LGTM in general. Just some small comments.

**Comment by mlafeldt** [comment]
Ah, you successfully confused me. Will review the other PR. 😄

**Pull request closed without merging**